mission_control/navigator_launch/launch/cameras.launch

```diff
@@ -0,0 +1,80 @@
+<launch>
+  <!-- This is the default pointgrey launch file with our id's and our names  -->
+
+  <arg name="front_camera_serial" default="15223362" />
+  <arg name="front_camera_calibrated" default="0" />
+  <arg name="down_camera_serial" default="10282860" />
+  <arg name="down_camera_calibrated" default="0" />
```

Comment:
You're the optometrist, your call. They do seem to be working ok. What was the main reason we decided to switch to 1394?